src/build/hibf/parse_chopper_pack_header.cpp

```diff
@@ -11,6 +11,8 @@
 #include <cassert>
 #include <seqan3/std/charconv>
 
+#include <chopper/prefixes.hpp>
+
 #include <raptor/build/hibf/bin_prefixes.hpp>
```

Comment:
Is this include outdated?

Reply:
Well it's technically not because the current code uses some of its variables but it should use `chopper/prefixes` in general. I'll make a PR for that